fabric_plugin/tasks.py

```diff
@@ -185,6 +186,18 @@ def download_resource(resource_path, target_path=None):
             proxy.close()
 
 
+def _create_process_config(process, operation_kwargs):
+    env_vars = operation_kwargs.copy()
+    if 'ctx' in env_vars:
+        del env_vars['ctx']
+    env_vars.update(process.get('env', {}))
+    for k, v in env_vars.items():
+        if isinstance(v, dict) or isinstance(v, list) or isinstance(v, set):
```

Comment:
there is a shorter syntax for isinstance, you can have the same condition like this:
`if isinstance(v, (dict, list, set))`